### DISCUSSION TITLE
[ask] recursion in closure function

### DISCUSSION THREAD

**Discussion by joysahimar** [vlang/v#19110] [Questions and Answers]
I have a fn recursion but in closure

Example Code
```v
fn reset(sum fn (a int, b int)) {
  sum(0, 1)
}

fn main() {
  sum := fn (a int, b int) {
    if a > 100 {
      reset(sum)
    }
  }
  sum(3, 4)
}
```
This code `error: sum must be added to the capture list for the closure to be used inside` 

and i'm try added to capture list but still error
```v
sum := fn [sum](a int, b int){...}
```
`error: undefined ident: sum`

**Comment by kimi996z** [comment]
it seems that this problem still exists now. I want to know if this problem is in vlang team's work plan.🤔
